static/js/components/Candidate.jsx

```diff
@@ -162,10 +163,12 @@ const Candidate = ({ route }) => {
               </AccordionSummary>
               <AccordionDetails>
                 <div className={classes.photometryContainer}>
```

Comment:
There's a style rule on `classes.photometryContainer` (from `useSourceStyles()`) that says `padding-bottom: 0.5rem`. I found that changing that to `padding: 0.5rem` solves the problem of the slider bar in the photometry plot being cut off slightly on the left edge. 

![Screenshot 2020-11-09 111154](https://user-images.githubusercontent.com/17696889/98566123-67575d80-227c-11eb-9327-5bb3e95d59c4.png)

Reply:
@kmshin1397 Thank you for helping with the review; feel free to push changes to this branch!

I think I know where the color mismatch came in---I re-oriented the checkboxes to save space.

Reply:
@stefanv Yeah, it's probably easiest for me to fix that since I wrote that part originally. I hard-coded some stuff in the JS logic for those that isn't responsive to the number of columns you set. I'll put that on my list of things to do today.

Reply:
> @kmshin1397 Thank you for helping with the review; feel free to push changes to this branch!
> 
> I think I know where the color mismatch came in---I re-oriented the checkboxes to save space.

Just to be clear: I meant I would not mind, not that you should!  I am happy to fix these myself later today.

Reply:
@stefanv No problem, I think I see all the errors - will push a patch soon